### PR TITLE
Adds fund branding organisation information with a bounty

### PR DESCRIPTION
**Pull request by SaptakS**
##### Description
- Allows funder to add fund branding organisation in the funding form
- Gitcoinbot comments the fund branding information as part of the comment.

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
funder form, gitcoinbot comment, model, migration

##### Refers/Fixes

Fixes: #1895

### CODE REVIEW

**Comment by SaptakS** [comment]
I haven't tested the gitcoinbot functionality whether it is working properly. @mbeacom can you help me with testing it and put this on staging?

Also, as mentioned in the issue, I was unable to find a way to comment in red in github. If anyone knows any alternative or feels the copy should be changed somewhere, please comment.

**Comment by owocki** [comment]
note that we'll have to make this compatible with https://github.com/gitcoinco/web/issues/1528 eventually

**Comment by owocki** [comment]
code LGTM; didnt test tho

**Comment by owocki** [comment]
note that mitch [was requesting that ECF branding be available on the bounty details / issue explorer pages too](http://bits.owocki.com/1j293I0W011d/Screen%20Shot%202018-08-20%20at%205.56.29%20PM.png) on his issue ...

not just in the github issue

File: app/dashboard/helpers.py [hunk 1]
```diff
@@ -407,6 +407,7 @@ def create_new_bounty(old_bounties, bounty_payload, bounty_details, bounty_id):
                 'contract_address': bounty_details.get('token'),
                 'network': bounty_details.get('network'),
                 'bounty_type': metadata.get('bountyType', ''),
+                'funding_organisation': metadata.get('fundingOrganisation', ''),
```
Comment (owocki) [inline app/dashboard/helpers.py]: this key shuld be added to `latest_old_bounty_dict` in this file

**Comment by SaptakS** [comment]
@owocki I thought we decided in the issue to go with option 2, which is adding to the github issue. I can add to bounty details and issue explorer page, but then I need what I need to add. Will the logo be different or just a copy or something else?

**Comment by owocki** [comment]
asked on https://github.com/gitcoinco/web/issues/1895#issuecomment-414711960 --- i think for now lets just deploy solution 2 and only do solution 1 if mitch replies affirmatively

**Comment by SaptakS** [comment]
Cool

File: app/dashboard/views.py [hunk 2]
```diff
@@ -929,6 +929,7 @@ def bounty_details(request, ghuser='', ghrepo='', ghissue=0, stdbounties_id=None
     if issue_url:
         try:
             bounties = Bounty.objects.current().filter(github_url=issue_url)
+            print(bounties)
```
Comment (thelostone-mc) [inline app/dashboard/views.py]: Do we need this ?

Comment (SaptakS) [reply app/dashboard/views.py]: Nope.

File: app/dashboard/models.py [hunk 3]
```diff
@@ -244,6 +244,7 @@ class Bounty(SuperModel):
     idx_project_length = models.IntegerField(default=0, db_index=True)
     idx_status = models.CharField(max_length=9, choices=STATUS_CHOICES, default='open', db_index=True)
     issue_description = models.TextField(default='', blank=True)
+    funding_organisation = models.TextField(null=True, blank=True)
```
Comment (mbeacom) [inline app/dashboard/models.py]: @SaptakS Will this ever be greater than 255 characters? If not, would you mind switching it to a `models.CharField(blank=True, max_length=255)` or whatever the max length might be (probably less...  100 maybe?) and dropping the `null=True` - Typically, you don't want to set `null=True` on `TextField` or `CharField`.

Following the change, can you delete and recreate the migration?

Comment (SaptakS) [reply app/dashboard/models.py]: Okay. I will put `default=''` in that case

**Comment by thelostone-mc** [comment]
@Saptak wanna just console.log the output and test this out ?

**Comment by mkosowsk** [comment]
@mbeacom @SaptakS  this is now live, correct?

When funding an issue I'm not sure where to look to fill out this info 🤔

Is it part of the `https://gitcoin.co/funding/new?url=https://github.com/EXAMPLE` page?

**Comment by mbeacom** [comment]
@mkosowsk It just went live.  If you refresh the new funding page, you should see the new input field.

**Comment by mkosowsk** [comment]
@mbeacom superb! Thanks for the help 👍 

Looks great @SaptakS 🥇

**Pull request merged**